stdlib/Markdown/docs/src/index.md

```diff
@@ -420,6 +420,10 @@ custom flavour of Markdown can be used, but this should generally be unnecessary
 Markdown.MD
 Markdown.@md_str
 Markdown.@doc_str
+Markdown.common
+Markdown.github
```

Comment:
I wonder how well supported this is -- I believe there have been quite many changes to Githubs flavor but no changes to the parser here. cc @mortenpi

Reply:
Honestly, I didn't even know there were flavor options for the parser. Maybe we could try to remove the option to customize this and see if anything breaks in the ecosystem? I.e. only allow the Julia flavor going forward, which I think is the only currently documented option?

Reply:
Sounds good.